optimizer-hints.md

```diff
@@ -191,6 +191,30 @@ EXPLAIN SELECT * FROM t WHERE EXISTS (SELECT /*+ SEMI_JOIN_REWRITE() */ 1 FROM t
 
 在上述例子中可以看到，在使用了 Hint 之后，TiDB 可以选择由表 `t1` 作为驱动表的 IndexJoin 的执行方式。
 
+### SHUFFLE_JOIN(t1_name [, tl_name ...])
+
+`SHUFFLE_JOIN(t1_name [, tl_name ...])` 提示优化器对指定表使用 Shuffle Join 算法，该 Hint 只在 MPP 模式下生效。例如：
```

Comment:
@Reminiscent I'm not sure whether our users are familiar with Shuffle Join. Do you think if it will be helpful to provide a reference link or explanation to shuffle join?

Ditto for broadcast join, 一阶段聚合 and 二阶段聚合

Reply:
在注意部分有提到，同时这部分也有提到需要在 `MPP 模式下生效`，了解这个概念应该就了解不同算法之间的区别。不了解的话，看注意中指向的文档应该也是能清楚的，感觉不需要进一步解释。